ios-sdk/TangramVC.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  TangramVC.swift
+//  ios-sdk
+//
+//  Created by Matt Smollinger on 7/12/16.
+//  Copyright © 2016 Mapzen. All rights reserved.
+//
+
+import UIKit
+import tangram_framework
+class TangramVC:  TGMapViewController{
```

Comment:
It's a little odd to mix up `TangramVC` with other names like `RoutingViewController`, they should be named following the same pattern. I personally like them explicit (leaving `ViewController`) for better understanding.